Update JetBrains Error Tracking tab information (#36750)
DataDog/documentation: content/en/ide_plugins/idea/error_tracking.md

Select an issue to display its details on the right-side of the tool window.

### Issue details
The issue details pane shows detailed information about the selected error, including the error status, the count, and a stack trace.

{{< img src="/ide_plugins/idea/error_tracking/issue_details.png" alt="Error Tracking details" style="width:100%;" >}}

Click the **View in Datadog** link to open the selected issue in Datadog.

Click on a file and line link in the stack trace to navigate to the code.

#### Fix with AI
When a local agent is detected (you have the Claude Code, Gemini, or Codex CLI installed locally), a **Fix with Local Agent** button will appear. Click this to open a new terminal and launch the local agent with a custom prompt from Datadog requesting the agent to fix the error.

#### Context tab

> The Context tab shows details about the issue including the service, first-seen and last-seen information, who the issue is assigned to, and the error count over time.

Reviewer: ```suggestion
The Context tab shows details about the issue including the service, first-seen and last-seen information, the issue's assignee, and the error count over time.
```

> {{< img src="/ide_plugins/idea/error_tracking/context_tab.png" alt="JetBrains Error Tracking Context tab" style="width:100%;" >}}

Reviewer: Consider scaling down (adjust `width`); the image is visually overwhelming at 100%.

Author: Decreased to 50%
